zigzag/inputs/examples/validation/sram_imc/dimc_validation/28nm/dimc_validation_subfunc.py

```diff
@@ -0,0 +1,178 @@
+import pdb
+from dimc_cost_model import *
+
+def dimc_cost_estimation(dimc, cacti_value):
+    unit_reg = UnitDff(dimc['unit_area'], dimc['unit_delay'], dimc['unit_cap'])
+    unit_area = dimc['unit_area']
+    unit_delay = dimc['unit_delay']
+    unit_cap = dimc['unit_cap']
+    input_channel = dimc['input_channel']
+    reg_input_bitwidth = dimc['reg_input_bitwidth']
+    input_bandwidth = input_channel * dimc['input_precision']
+    output_bandwidth_per_channel = dimc['output_precision']
+    """
+    multiplier array for each output channel
+    """
+    if dimc['booth_encoding'] == True:
+        mults = MultiplierArray(vdd=dimc['vdd'],input_precision=int(dimc['multiplier_precision']),number_of_multiplier=input_channel*dimc['input_precision']/2, unit_area=unit_area, unit_delay=unit_delay, unit_cap=unit_cap)

```

Comment:
I don't understand why there's use of a "MultiplierArray" in here, I thought you had defined a "DimcArray" which is basically the equivalent of the "MultiplierArray"?

Reply:
Here is a standalone class used solely for validation. The validation scripts are separate from the classes within zigzag, so the validation result will not be affected by any change in zigzag in the future.